core/domain/user_domain.py

```diff
@@ -991,18 +1128,23 @@ def __init__(
         self.can_review_questions = can_review_questions
         self.can_submit_questions = can_submit_questions
 
-    def can_review_at_least_one_item(self):
+    def can_review_at_least_one_item(self) -> bool:
         """Checks whether user has rights to review at least one item.
 
         Returns:
             boolean. Whether user has rights to review at east one item.
         """
-        return (
+        # Due to comparison between different types (List[str] and bool) MyPy
```

Comment:
I would say, instead: "Note that XXX and YYY are list(str), so we need the bool cast to ensure that the return value is a boolean."

That is, we are not doing this just for mypy; we are doing it for the integrity of the method per the contract specified in the function description.

Reply:
Comment changed!
Done.